lib/gettext.js

```diff
@@ -42,10 +42,17 @@ var i18n = function (options) {
    // sprintf equivalent, takes a string and some arguments to make a computed string
    // eg: strfmt("%1 dogs are in %2", 7, "the kitchen"); => "7 dogs are in the kitchen"
    // eg: strfmt("I like %1, bananas and %1", "apples"); => "I like apples, bananas and apples"
+   // NB: removes msg context if there is one present
    var strfmt = function (fmt) {
-     var args = arguments;
+      var args = arguments;
 
-     return fmt
+      // if there is context, remove it
+      if (fmt.indexOf(_ctxt_delimiter) !== -1) {
```

Comment:
I'm just curious, why not just handle this in the translate function before calling strfmt?

Reply:
Good point: I first handled that in the translate function, but I noticed it had a quite important footprint since there are some conditions with singular and plurals. It felt easier to handle it once with a simple string in the `strfmt` method that was called anyway, even if it doesn't _really feels right_ to be there.

Since I'm making a lot to apologise, I'll try to make something better before merging :p